st.clab.yml

```diff
@@ -116,6 +116,39 @@ topology:
         GF_AUTH_ANONYMOUS: "true"
       group: "10"
 
+    ### LOGGING STACK ###
+    syslog:
+      kind: linux
+      mgmt-ipv4: 172.80.80.44
+      image: linuxserver/syslog-ng:latest
+      binds:
+        - configs/syslog/:/config
+        - configs/syslog/log:/var/log
+      env:
+        PUID: 0
+        PGID: 0
+
+    promtail:
+      kind: linux
+      mgmt-ipv4: 172.80.80.45
+      image: grafana/promtail:2.7.4
+      binds:
+        - /var/log/:/var/log
```

Comment:
@KTodts I think it makes sense to use local dir under promptail dir as a log directory, rather than using `/var/log`. To keep things local to the lab directory

Reply:
Removing this as it's not needed